continuous_integration/scripts/build-legate-wheel

```diff
@@ -21,7 +21,7 @@ build_legate_wheel() {
 
     local cmake_args=(${CMAKE_ARGS:-});
     cmake_args+=("-DFIND_LEGATE_CORE_CPP=ON");
-    cmake_args+=("-Dlegate_core_ROOT=$REPO_DIR/legate/build");
+    cmake_args+=("-Dlegate_core_ROOT=$REPO_DIR/build");
```

Comment:
This typo was causing the legate cpp from the first build not being found and the second build was actually building everything from scratch. Because the second build has only the extra cmake arguments required on the top of the cpp build, it was building a wrong configuration causing illegal instruction issues and entirely skipping ucx tests.